angular-client/src/components/battery/battery.component.css

```diff
@@ -0,0 +1,6 @@
+.battery {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
```

Comment:
why absolute

Reply:
i experimented with other positionings and considering this is a component that will be used in different settings I think absolute works best